setup.py

```diff
@@ -195,12 +187,10 @@ def _run_domain_js(self, domain):
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
```

Comment:
We should provide:
`Programming Language :: Python :: 3 :: Only`

Reply:
done